src/core_plugins/kibana/public/dashboard/filter_utils.js

```diff
@@ -14,7 +15,7 @@ export class FilterUtils {
    * (e.g. goes in the query input bar), false otherwise (e.g. is in the filter bar).
    */
   static isQueryFilter(filter) {
-    return filter.query && filter.query.query_string && !filter.meta;
+    return filter.query && !filter.meta;
```

Comment:
Yes, that is fixed by this. But it also affects this PR... the default query is now a match all, which wouldn't have met the condition `filter.query.query_string`.